_episodes/03-technical.md

```diff
@@ -101,6 +101,38 @@ are modifying.
 > screenshared.
 {: .challenge}
 
+## Configuring and Sync Forks
+
+Forks are often used in open source development on GitHub 
+to make changes to a project without affecting the original repository. 
+You must first configure a remote that points to the upstream repository in Git.
+Then, you can sync and fetch updates from or 
+submit changes to the original repository with pull requests. 
+
+To **configure a remote** for a fork, follow these commands:
+
+~~~
+$ git remote -v
+$ git git remote add upstream https://github.com/ORIGINAL_OWNER/ORIGINAL_REPOSITORY.git
```

Comment:
`git git` typo

Reply:
You will need to fetch after adding the remote, otherwise the remote branch will not exist and you won't be able to merge.

Reply:
Thanks, @remram44. I corrected the `git git` typo. 

I have a` git fetch` command on line 125. Is that sufficient? It is the next command, but I have it in the following block of code.